cdc/redo/manager.go

```diff
@@ -533,20 +533,35 @@ func (m *ManagerImpl) onResolvedTsMsg(tableID model.TableID, resolvedTs model.Ts
 func (m *ManagerImpl) bgUpdateLog(ctx context.Context, errCh chan<- error) {
 	logErrCh := make(chan error, 1)
 	handleErr := func(err error) {
+		log.Warn("redo manager flushLog fail",
+			zap.String("namespace", m.changeFeedID.Namespace),
+			zap.String("changefeed", m.changeFeedID.ID),
+			zap.Error(err))
 		select {
 		case logErrCh <- err:
 		default:
 		}
 	}
 
+	log.Info("redo manager bgUpdateLog is running",
```

Comment:
too many logs will appear in the log files

Reply:
There will be one this log per changefeed. So I guess it's acceptable.